docs/source/about/advanced_usage.rst

```diff
@@ -1077,6 +1130,96 @@ nvenc_realtime_hags
 
       nvenc_realtime_hags = enabled
 
+nvenc_h264_hevc_ref_frames
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+   Default value for H.264 and HEVC reference frames in DPB (decoder picture buffer).
+   Override default sunshine value in situations when moonlight doesn't request one explicitly.
+   Higher values can provide larger pool for reference frame invalidation on unstable networks, but specific decoders may not handle it well depending on video resolution.
+
+   .. Note:: This option only applies when using NVENC `encoder`_.
+
+   .. Note:: Doesn't apply to AV1 since it uses predetermined value as part of specification.
+
+   .. Warning:: Can lead to (or remedy) decoding artifacts.
+
+   .. Caution:: Applies to Windows only.
+
+**Default**
+   Variable is unset and sunshine will pick whatever it thinks is the best.
```

Comment:
So the default is not 5?

Generally, placeholders in the config ui match the default.

Reply:
Current default is constant 5, but I can't guarantee it will stay as single constant value in the future and not change dynamically depending on let's say video resolution.

Reply:
Okay, if it changes in the future to an auto mode this can be adjusted accordingly.

For now, I think this should be `5`. Also, as long as the config UI uses the correct defaults, then it allows us to modify the default when a user updates. As long as they are using our old default, they will also start using our new default.